webapp/portlet/src/main/webapp/vue-apps/spaces-list/components/ExoSpacesToolbar.vue

```diff
@@ -9,7 +9,7 @@
       <v-btn
         v-else
         id="addNewSpaceButton"
-        class="btn btn-primary px-0 addNewSpaceButton"
+        class="btn btn-primary px-0 mx-2 mx-sm-0 addNewSpaceButton"
```

Comment:
the used breakpoint `sm` seems different from the definition of `isMobile` property (`mdAndDown`), please make sure to use the same breakpoint for mobile breakpoint definition.

Reply:
cc @MayTekayaa